lib/widgets/task_app_header.dart

```diff
@@ -0,0 +1,220 @@
+import 'package:confetti/confetti.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fokus/logic/timer/timer_cubit.dart';
+import 'package:fokus/model/currency_type.dart';
+import 'package:fokus/services/app_locales.dart';
+import 'package:fokus/services/ticker.dart';
+import 'package:fokus/utils/theme_config.dart';
+import 'package:fokus/widgets/app_header.dart';
+
+import 'chips/attribute_chip.dart';
+import 'large_timer.dart';
+
+class TaskAppHeader extends StatefulWidget with PreferredSizeWidget {
+	final double height;
+	final String title;
+	final String text;
+	final Widget appHeaderWidget;
+	final String helpPage;
+	final Function breakPerformingTransition;
+	final bool isBreak;
+
+	TaskAppHeader({Key key, @required this.height, this.title, this.text, this.appHeaderWidget, this.helpPage,this.breakPerformingTransition, this.isBreak}) : super(key: key);
+
+	@override
+	Size get preferredSize => Size.fromHeight(height);
+
+  @override
+  State<StatefulWidget> createState() => TaskAppHeaderState();
+}
+
+class TaskAppHeaderState extends State<TaskAppHeader> with TickerProviderStateMixin {
+	AnimationController _buttonController;
+	ConfettiController _confetti;
+	bool isBreakNow;
+	CrossAxisAlignment alignment = CrossAxisAlignment.start;
+	Animation<Offset> _offsetAnimation;
+	AnimationController _slideController;
+	final String _pageKey = 'page.childSection.taskInProgress';
+
+  @override
+  Widget build(BuildContext context) {
+		return Column(
```

Comment:
You could break that method up into at least the actual appBar, timer and button